test/__snapshots__/index.spec.js.snap

```diff
@@ -4,9 +4,8 @@ exports[`lintStaged should not output config in non verbose mode 1`] = `""`;
 
 exports[`lintStaged should ouput config in verbose mode 1`] = `
 "
-LOG 
-Running lint-staged with the following config:
-{
+LOG Running lint-staged with the following config:
+         {
```

Comment:
That's an odd indentation here

Reply:
You're right, sorry I missed it. Can be fixed with this:
```js
console.log(`Running lint-staged with the following config: \n${stringifyObject(config)}`)
```
Is this ok?

Reply:
Okay with me!